AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -3175,7 +3175,7 @@ uhrzeit.org###cookieContraption
 shopdisney.de#$##disney-explicit-consent-prompt { position: absolute!important; left: -2000px!important; } 
 mdv.de###BorlabsCookieBoxWrap
 dm.de###overlays > div[data-dmid="cookiebar-dm-overlay"]
-vegascreativesoftware.com,magix.com###cookieConsentMessage
+dl03.magix.net,vegascreativesoftware.com,magix.com###cookieConsentMessage
```

Comment:
Please add this rule to tweaks.txt 
`dl03.magix.net#$#header { margin-top: 0!important; }`